copydb/cmd/main.go

```diff
@@ -121,7 +120,7 @@ func main() {
 		return
 	}
 
-	if resumeState == nil {
+	if config.StateToResumeFrom == nil {
```

Comment:
`resumeState` was previously only set if we passed in a path for the statedump file. We want to be able to pass in the state dump directly in the config through the field `StateToResumeFrom` as well.